features/ssh/exec.config

```diff
@@ -1,5 +1,34 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
+set -x
```

Comment:
```suggestion
```

Reply:
thanks, forgot to remove after debugging